airflow/dag_processing/manager.py

```diff
@@ -1187,6 +1188,21 @@ def _collect_results_from_processor(self, processor) -> None:
             span.set_attribute("import_errors", count_import_errors)
             if count_import_errors > 0:
                 span.set_attribute("error", True)
+                import_errors = (
+                    session.query(ParseImportError)
```

Comment:
Let's use sqlalchemy 2 query here

Reply:
> Let's use sqlalchemy 2 query here

Hi @ephraimbuddy , how am I be able to use it? Any suggestion code snippet would be nice.